proof_test.go

```diff
@@ -893,3 +894,52 @@ func TestVerifyNamespace_ShortAbsenceProof_Invalid(t *testing.T) {
 		})
 	}
 }
+
+func TestProtoToProof_InclusionProof(t *testing.T) {
+	pbProof := pb.Proof{
+		0,
+		1,
+		[][]byte{bytes.Repeat([]byte{1}, 10)},
+		nil,
+		true,
+	}
```

Comment:
golangci-lint complains here. It wants keys for each field

```suggestion
	pbProof := pb.Proof{
		Start:                 0,
		End:                   1,
		Nodes:                 [][]byte{bytes.Repeat([]byte{1}, 10)},
		LeafHash:              nil,
		IsMaxNamespaceIgnored: true,
	}
```